src/api.routes.js

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Since all these are constants, I would suggest we have them all consolidated in one file `constants.js`. A separate module would make much more sense. i.e.`/src/constants/index.js`